frontend/ui/widget/touchmenu.lua

```diff
@@ -595,6 +595,9 @@ function TouchMenu:updateItems()
     table.insert(self.item_group, self.footer)
     -- @translators %1 is the current page. %2 is the total number of pages. In some languages a good translation might need to reverse this order, for instance: "Total %2, page %1".
     self.page_info_text.text = util.template(_("Page %1 of %2"), self.page, self.page_num)
+    if self.page == self.page_num then
+      self.page_info_text:setText("");
```

Comment:
Not sure why, but in the top menu, when there are multiple pages, I don't see anymore `Page P of N`, I always see blank between arrows (eg, with crowded menu `Gear > Tap & gestures > Gesture manager > Multiswipe`)

Reply:
Can be fixed by applying the same solution as menu.lua if page_num > 1, will update code